config/locales/en.yml

```diff
@@ -92,6 +92,8 @@ en:
     1400:
       103: 'Channel Islands duty'
       105: 'Channel Islands duty'
+    1080:
+      103: 'UK-CD Customs Union'
```

Comment:
... and is we should we also include the code `105` (which stands for “Non preferential duty under authorised use”)

Reply:
The answer to this question is in the database:

```ruby
Measure.where(geographical_area_id: '1080').distinct(:measure_type_id).pluck(:measure_type_id)
=> ["103"]
```

Reply:
I think it would be harmless to add this for 105 and would avoid us falling into another change later down the line